src/main/java/seedu/address/model/meeting/UniqueMeetingList.java

```diff
@@ -97,6 +98,38 @@ public void setMeetings(List<Meeting> meetings) {
         internalList.setAll(meetings);
     }
 
+    /**
+     * Replaces a participant {@code target} in all meetings it is
+     * participating in with {@code editedParticipant}.
+     *
+     * @param target the participant to be updated
+     * @param editedParticipant the updated participant
+     */
+    public void setMeetingParticipant(Participant target, Participant editedParticipant) {
```

Comment:
Maybe it would be better to use a different name for the method here since the same method name is being used across different classes? 🤔

Reply:
You're right, it can get a little confusing. Thanks again for pointing it out! 😄